### DISCUSSION TITLE
Adding pl.yaml file to i18n

### DISCUSSION THREAD

**Discussion by eduakcja** [adityatelange/hugo-PaperMod#349] [Ideas]
Will it be possible to add a Polish translation to i18n files. I volunteer to translate it.

**Comment by kdkasad** [comment]
You're welcome to make the changes and then open a PR to get them merged.

**Comment by meleu** [comment]
btw, how is it possible to make use of those translated files?
(I also posted a question [here](https://github.com/adityatelange/hugo-PaperMod/discussions/843))
